.gitignore

```diff
@@ -1,3 +1,4 @@
 /venv/*
 blog.db
-run.pyc
\ No newline at end of file
+run.pyc
+form.pyc
```

Comment:
rather than update this everytime you add a new file you should add wildcard ignores here 

`*.pyc`

Reply:
done

Reply:
done